ui/insight/src/global.d.ts

```diff
@@ -0,0 +1,71 @@
+// Extend the global lichess object with a function defined in
+// public/javascripts/insight-refresh.js
+interface Lichess {
+  refreshInsightForm(): void;
+}
+
+/* Type definitions for multiple-select.js */
+
+// multiple-select.js uses jquery, but typescript thinks $ is Cash, not jquery.
+// So as a hack, we extend the Cash interface knowing that at runtime, it will
+// actually be a jquery object.
```

Comment:
I wasn't sure if there was a way to import Lichess types without making cash conflict with jquery. Cash functions and types are close enough to jquery that no casting is required, so for now I'm using cash types on jquery values. It works, but it feels bad. Maybe there's a better way?